File: api/src/main/java/net/asch/bulkit/api/BulkIt.java
package net.asch.bulkit.api;

import net.minecraft.resources.ResourceLocation;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class BulkIt {
    static public final String ID = "bulkit";
    static private final Logger LOGGER = LogManager.getLogger();

    static public ResourceLocation location(String path) {
        return ResourceLocation.fromNamespaceAndPath(ID, path);
    }

    static public void logInfo(String msg) {
        LOGGER.info(msg);
    }

    static public void logDebug(String msg) {
        LOGGER.debug(msg);
    }
}

File: src/api/kotlin/net/asch/bulkit/api/BulkIt.kt
package net.asch.bulkit.api

import net.minecraft.resources.ResourceLocation
import org.apache.logging.log4j.LogManager

object BulkIt {
    const val ID = "bulkit"
    private val LOGGER = LogManager.getLogger()

    fun location(path: String): ResourceLocation = ResourceLocation.fromNamespaceAndPath(ID, path)
    fun logInfo(msg: String) = LOGGER.info(msg)
    fun logDebug(msg: String) = LOGGER.debug(msg)
}